bi/pkg/specs/battery_spec.go

```diff
@@ -25,10 +25,19 @@ func (s *InstallSpec) GetBatteryConfigField(typ, field string) (any, error) {
 	}
 	return cfg, nil
 }
+
 func (s *InstallSpec) GetCoreNamespace() (string, error) {
 	ns, err := s.GetBatteryConfigField("battery_core", "core_namespace")
 	if err != nil {
 		return "", err
 	}
 	return ns.(string), nil
 }
+
+func (s *InstallSpec) GetCoreUsage() (string, error) {
```

Comment:
The usage is pulled a half dozen times or so already to wrap it up in a func for reuse.